Cargo.toml

```
@@ -50,7 +50,7 @@ serde = { version = "1.0", default-features = false, optional = true, features =
 # The original packed_simd package was orphaned, see
 # https://github.com/rust-lang/packed_simd/issues/303#issuecomment-701361161
 packed_simd = { version = "0.3.4", package = "packed_simd_2", features = ["into_bits"], optional = true }
-zeroize = { version = ">=1, <1.4", default-features = false }
+zeroize = { version = "1.5.5", default-features = false }
```

There's no need for it to be this restrictive, and making it restrictive creates interoperability problems.

```suggestion
zeroize = { version = "1", default-features = false }
```

---

Makes sense, updated